server/webapp/WEB-INF/rails/webpack/views/components/flash_message/index.scss

```diff
@@ -23,11 +23,16 @@
   border-radius: $global-border-radius;
   margin:        0 0 10px 0;
   position:      relative;
-
+  overflow:      auto;
+  max-height:    200px;
   p {
     margin: 0;
   }
-
+  pre {
```

Comment:
`margin: 0` to reset <pre> element's default margin

Reply:
Do not use global styles like these `p` and `pre` will apply globally. Only apply styles on classes.

Reply:
@ketan But these styles are under class `.callout`, so they would not be applied globally.